lib/configuration.js

```diff
@@ -87,7 +87,7 @@ var Configuration = function(givenConfig, logger) {
    * @type {Boolean}
    */
   this._shouldReportErrorsToAPI = env.NODE_ENV === 'production';
-  if (!this._shouldReportErrorsToAPI) {
```

Comment:
In that case, could we put together a dummy logger for the tests and remove the check?